### ISSUE TITLE
Final review

### ISSUE THREAD

**Issue by damianooldoni** [riparias/union-list#13]
To be reviewed:

- [ ] the dates in field `entry_into_force` in raw data (tsv file) some are in the future, e.g. for _Celastrus orbiculatus_ is this field set to 2027-08-02, while for _Fundulus heteroclitus_ and_Xenopus laevis_ is 2024-08-02. I see also that in 2022 we have two very near dates: `2022-07-12` and `2022-08-02`. Maybe is correct, but still worth a final check
- [ ] the link to the most actual consolidated pdf version (I modified the link to point the 2022 list, please @timadriaens double check this) in [Workflow section](https://github.com/riparias/union-list#workflow ) of README.
- [ ] The webpage with English common names is outdated. I couldn't find any other similar webpage. The only link with common names is the PDF brochure to download on [circabc](https://circabc.europa.eu/ui/group/4cd6cb36-b0f1-4db4-915e-65cd29067f49/library/b71ec258-a482-41cf-8ab3-5d443b035c5f/details) but I find not a good enough link.
- [ ] [metadata](https://docs.google.com/document/d/1VDxqzyQxMsEF3kZDu9WSRAwuB2PnkG03AOd745cN8_Y/edit#). I went through the document and seems good, but still, @timadriaens, we need a sentence mentioning the issues encountered while harvesting data from EASIN API, see #9 (+ #10), I think, to justify our approach.

**Comment by timadriaens** [comment]
- [ ] future dates are correct, this is because of prolonged transitionary measures foreseen in the Regulation. Still, I would already list those species as Member States are already working on them.
- [ ] [Link](https://eur-lex.europa.eu/legal-content/EN/TXT/PDF/?uri=CELEX:02016R1141-20220802&from=EN) to the consolidated list in the README is fine
- [ ] Common names document: the [IAS webpage of IUCN](https://www.iucn.org/our-work/topic/invasive-alien-species) was refurbished recently (and the resources page for the EU Reg you find [here](https://www.iucn.org/our-work/topic/invasive-alien-species/invasive-alien-species-additional-external-resources)) and doesn't seem to have a link to the file anymore. I filed a request with IUCN colleagues.
- [ ] I have mentioned EASIN in the additional information sentence. We will communicate with them after we published the checklist to see if we can indeed in the future harvest it from their api instead (but for now I would refrain from mentioning any data gaps there).

**Comment by timadriaens** [comment]
I had answer for IUCN, here are the files with common names:

https://circabc.europa.eu/ui/group/4cd6cb36-b0f1-4db4-915e-65cd29067f49/library/2ed2ee2a-730f-4583-a3ac-bb2a1815ad6a/details

https://circabc.europa.eu/ui/group/4cd6cb36-b0f1-4db4-915e-65cd29067f49/library/1ac00d12-613b-447c-ab91-3016af071bcf/details

https://circabc.europa.eu/ui/group/4cd6cb36-b0f1-4db4-915e-65cd29067f49/library/fad036d3-e2df-4adb-9c7a-7b9593a4c2f8/details

IUCN have not yet merged those files...

**Comment by damianooldoni** [comment]
@timadriaens: I will add the link to those three files in README. Could you please check that these names are the same as the ones in our tsv? Or did you already done it? Thanks.

**Comment by timadriaens** [comment]
should we not include the common names in other languages? If you want I can do that.

**Comment by damianooldoni** [comment]
yes, we can. If you do, I am happy to check and map it further to DwC 👍  I also noticed that the names in the spreadsheets are not always correct as we think they should be, e.g. capital letters in French and English common names should be removed unless referring to a country, sin't? I also see that we are not always applying this rule: see e.g. "Pumpkinseed" which probably shoul be "pumpkinseed"? I let this issue to you as it is not my domain of expertise 😄

**Comment by damianooldoni** [comment]
We can discuss this further if you need in #14. We leave this issue for final review.

**Comment by damianooldoni** [comment]
Data and metadata have been reviewed.
